catalogue/webapp/pages/work.js

```diff
@@ -180,6 +180,7 @@ const WorkPage = ({ work }: Props) => {
       type={'website'}
       url={`https://wellcomecollection.org/works/${work.id}`}
       imageUrl={iiifImage({size: '800,'})}
+      siteSection='images'
```

Comment:
Is this really the right name?
Could we call it catalogue?

Reply:
Could we update it at the point we update the nav label? If not, we'll have three independent words for the same section!

Reply:
💯